### ISSUE TITLE
Sort by name

### ISSUE THREAD

**Issue by rafariva** [sirmordred/WhatsCloud-Tracker#5]
Exporting a group chat, it shows by name but should be sort by it

**Comment by sirmordred** [comment]
Good idea, will hopefully be implemented in next version of WhatsCloud 👍  marking it as "Enhancement"

Thanx
